src/python/pants/backend/docker/lint/hadolint/rules.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Oh didn't realize this is still under an experimental backend. @benjyw do you think it should be still? Now that we have Hadolint, Docker support is a Real Thing, more than just `./pants tailor`. For example, our Shell support is only linters + a test runner.

Reply:
Let's worry about that later. We still have to add the core functionality.

Reply:
Okay. @kaos can you please rename this folder to be experimental then? Note that this backend doesn't work without first activating the core Docker backend.

Reply:
Looking at terraform, only the register file is under `experimental`.. same here?

Reply:
Yeah that's fine, given how stable we expect this code itself to be. Thanks!